rockcraft.yaml

```diff
@@ -1,7 +1,7 @@
 name: identity-platform-login-ui
 
 base: bare
-build-base: ubuntu:22.04
+build-base: ubuntu@22.04
```

Comment:
can we separate this into another commit?
or simply drop it and we ll chuck it in the main branch and u can rebase

Reply:
I factored this into a dedicated commit.